### PR TITLE
Update Simplified Chinese trsl

### PR DESCRIPTION
**Pull request by ChuijkYahus**
Done on the latest version. Manual problems are fixed.
Also fixed the invalid links in Motorboat entry.

Asphalt Stairs' name won't display properly.

https://github.com/CFPAOrg/Minecraft-Mod-Language-Package/pull/2659

### CODE REVIEW

**Comment by TwistedGate** [comment]
> Asphalt Stairs' name won't display properly.

Any specific reason?

**Comment by ChuijkYahus** [comment]
> > Asphalt Stairs' name won't display properly.
> 
> Any specific reason?

Possibly because of a typo in key name (namely `^`), it would only display as key name in-game.

File: src/main/resources/assets/immersivepetroleum/lang/zh_cn.json [hunk 1]
```diff
@@ -11,41 +11,61 @@
 	
 	"block.immersivepetroleum.asphalt": "沥青混凝土",
 	"block.immersivepetroleum.asphalt_slab": "沥青混凝土台阶",
-	"block.immersivepetroleum.asphalt_stairs": "沥青混凝土楼梯",
+	"block.immersivepetroleum.asphalt_stair^": "沥青混凝土楼梯",
```
Comment (TwistedGate) [inline src/main/resources/assets/immersivepetroleum/lang/zh_cn.json:14]: Just remove `^` then, it isnt in the original either. Don't know why you guys put it in there :D

**Comment by TwistedGate** [comment]
Ah i see.. don't know how that got in there honestly..

just remove it and it should work.

**Comment by TwistedGate** [comment]
a92d99d5c570a31cf1c5ff6e618158e0fb2204d4 Removed it from the original aswell now, so once this PR has it removed aswell i'll merge it.

**Comment by ChuijkYahus** [comment]
Removed it, should be OK for merging i think.

**Pull request merged**